apps/examples/testcase/le_tc/kernel/itc_environ.c

```diff
@@ -0,0 +1,111 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
```

Comment:
Please check the year

Reply:
changes done in all files as per your comments